CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# next -- ????.??.??
+
+* Backport `Eq` and `Ord` instances for `Generically1`, which were introduced
+  in `base-4.18.0.0`.
```

Comment:
I'd add add dependency to `base-orphans` with these instances to always "pollute" / provide them from `GHC.Generics.Generically`.

Reply:
Good idea. In that case, I will park this PR for now and update it after haskell-compat/base-orphans#65 has been released to Hackage.

Reply:
I've updated the PR to bring `Data.Orphans` into scope when needed.